Explorer/Assets/DCL/UI/ButtonWithAnimationView.cs

```diff
@@ -34,21 +34,21 @@ private void OnDisable()
             ButtonAnimator.enabled = false;
         }
 
-        private void OnClick()
-        {
-            ButtonAnimator.SetTrigger(AnimationHashes.PRESSED);
-            UIAudioEventsBus.Instance.SendPlayAudioEvent(ButtonPressedAudio);
-        }
-
         public void OnPointerEnter(PointerEventData eventData)
         {
-            ButtonAnimator.SetTrigger(AnimationHashes.HOVER);
+            ButtonAnimator.SetTrigger(UIAnimationHashes.HOVER);
             UIAudioEventsBus.Instance.SendPlayAudioEvent(ButtonHoverAudio);
         }
 
         public void OnPointerExit(PointerEventData eventData)
         {
-            ButtonAnimator.SetTrigger(AnimationHashes.UNHOVER);
+            ButtonAnimator.SetTrigger(UIAnimationHashes.UNHOVER);
+        }
+
+        private void OnClick()
+        {
+            ButtonAnimator.SetTrigger(UIAnimationHashes.PRESSED);
+            UIAudioEventsBus.Instance.SendPlayAudioEvent(ButtonPressedAudio);
```

Comment:
Ideally you should be able to [SerializeField] a ScriptableObject which holds the instance of the `UIAudioEventsBus` so we dont nest singletons across the solution, which might become a pain if not managed properly. (refactor not required on this pr)

Reply:
Yes, we need to refactor it in the not so distant future